cmd/shortener/server/flags.go

```diff
@@ -0,0 +1,28 @@
+package server
+
+import (
+	"flag"
+	"fmt"
+	"github.com/Taboon/urlshortner/cmd/shortener/config"
+	"os"
+)
+
+func parseFlags() {
```

Comment:
It would be better if you return `error`

Reply:
Redid the output of the function